schemas/core/leg.json

```diff
@@ -94,14 +86,20 @@
               "$ref": "components/units-geo.json#/definitions/polyline"
             }
           }
+        },
+        "tspProduct": {
+          "$ref": "http://maasglobal.com/core/booking-option.json#/definitions/tspProduct"
+        },
+        "productOption": {
+          "description": "Index of the productOption used in the itinerary's productOptions",
+          "type": "integer",
+          "minValue": 0
         }
       },
-      "required": ["from", "to", "mode", "startTime", "endTime", "legGeometry"],
-      "additionalProperties": false
```

Comment:
Remove this. Do not know it there will be any implication.

Reply:
Is the `additionalProperties' a specific problem that we are fixing now? If not, and if it has been there for a while, then maybe leave it as it is?

Reply:
Leaving it as is